samples/snippets/src/main/java/aiplatform/BatchCreateFeaturesSample.java

```diff
@@ -0,0 +1,104 @@
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featurestoreId = "YOUR_FEATURESTORE_ID";
+    String entityTypeId = "YOUR_ENTITY_TYPE_ID";
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+    int timeout = 300;
+    batchCreateFeaturesSample(project, featurestoreId, entityTypeId, location, endpoint, timeout);
+  }
+
+  static void batchCreateFeaturesSample(String project, String featurestoreId, String entityTypeId,
+      String location, String endpoint, int timeout)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    FeaturestoreServiceSettings featurestoreServiceSettings =
+        FeaturestoreServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (FeaturestoreServiceClient featurestoreServiceClient =
+        FeaturestoreServiceClient.create(featurestoreServiceSettings)) {
+
+      List<CreateFeatureRequest> createFeatureRequests = new ArrayList<>();
+
+      Feature titleFeature = Feature.newBuilder().setDescription("The title of the movie")
+          .setValueType(ValueType.valueOf("STRING")).build();
+      Feature genresFeature = Feature.newBuilder().setDescription("The genres of the movie")
+          .setValueType(ValueType.valueOf("STRING")).build();
+      Feature averageRatingFeature = Feature.newBuilder()
+          .setDescription("The average rating for the movie, range is [1.0-5.0]")
+          .setValueType(ValueType.valueOf("DOUBLE")).build();
+
+      createFeatureRequests.add(
+          CreateFeatureRequest.newBuilder().setFeature(titleFeature).setFeatureId("title").build());
+
+      createFeatureRequests.add(CreateFeatureRequest.newBuilder().setFeature(genresFeature)
+          .setFeatureId("genres").build());
+
+      createFeatureRequests.add(CreateFeatureRequest.newBuilder().setFeature(averageRatingFeature)
+          .setFeatureId("average_rating").build());
+
+      BatchCreateFeaturesRequest request = BatchCreateFeaturesRequest.newBuilder()
+          .setParent(EntityTypeName.of(project, location, featurestoreId, entityTypeId).toString())
+          .addAllRequests(createFeatureRequests).build();
+
+      OperationFuture<BatchCreateFeaturesResponse, BatchCreateFeaturesOperationMetadata> future =
+          featurestoreServiceClient.batchCreateFeaturesAsync(request);
+      System.out.format("Operation name: %s%n", future.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      BatchCreateFeaturesResponse batchCreateFeaturesResponse =
+          future.get(timeout, TimeUnit.SECONDS);
```

Comment:
just to confirm, is the behavior here to wait the 300 sec and timeout otherwise. If there is a response available, it's returned.

Reply:
yes, if the response is not received in 300 sec it will throw timeout